symplyphysics/definitions/amount_energy_from_mass_and_temperature.py

```diff
@@ -0,0 +1,30 @@
+from symplyphysics import (
+    symbols, Eq, pretty, solve, Quantity, units,
+    validate_input, validate_output, expr_to_quantity
+)
+
+# Description
+# Amount of energy for body heat Q = C * m * (t2 - t1)
+# where:
+# C - specific heat capacity
```

Comment:
Add high level description of this parameter. Add wiki link if it cannot be easily explained.